Assets/GothicVR/Scripts/Creator/NpcCreator.cs

```diff
@@ -65,12 +67,8 @@ public static void Wld_InsertNpc(int npcInstance, string spawnpoint)
 
             newNpc.GetComponent<Properties>().npc = pxNpc;
 
-            if (PhoenixBridge.npcRoutines.TryGetValue(pxNpc.npcPtr, out List<RoutineData> routines))
-            {
-                initialSpawnpoint = PhoenixBridge.World.waypoints
-                    .FirstOrDefault(item => item.name.ToLower() == routines.First().waypoint.ToLower());
-                newNpc.GetComponent<Routine>().routines = routines;
-            }
+            var initialSpawnpointName = newNpc.GetComponent<Routine>().routines.First().waypoint;
```

Comment:
Is it always delivering a value? Or can .routines().First() be empty?
If so, then please add an if statement again.

Reply:
I changed it, but I think its superflous anyways, since at the top of the function there's already another intialSpawnpoint calculation. It works differently though and I dont know where the input comes from, so I cant be completely sure.